phase1/gce/configure-vm.sh

```diff
@@ -4,12 +4,14 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-ROLE=$(curl \
-  -H "Metadata-Flavor: Google" \
-  "metadata/computeMetadata/v1/instance/attributes/k8s-role")
-BUCKET=$(curl \
-  -H "Metadata-Flavor: Google" \
-  "metadata/computeMetadata/v1/instance/attributes/k8s-deploy-bucket")
+get_metadata() {
```

Comment:
Just had quick thought... Would fetching and parsing these from Ansible be reasonable in terms of phase separation?